### PR TITLE
TN-2022 Update Backend Translations

### PR DESCRIPTION
**Pull request by ivan-c**
* Update backend translations (`flask download-translations`)
  * Add new email footers
  * Remove translations for strings that are no longer present

### CODE REVIEW

File: portal/translations/es_ES/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -2276,186 +2294,190 @@ msgid "America/New York"
 msgstr "Estados Unidos/Nueva York"
 
 #: https://github.com/uwcirg/truenth-portal/tree/develop/portal/templates/profile/profile_macros.html#L932
+msgid "America/Sao Paulo"
+msgstr ""
```
Comment (mcjustin) [inline portal/translations/es_ES/LC_MESSAGES/messages.po]: @ivan-c  just noting this new msgid and it's empty msgstr

File: portal/translations/es_ES/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -2591,665 +2613,669 @@ msgstr "No se encontró ninguna identidad"
...
-# Organization: Winship Cancer Institute Emory University
-msgid "Winship Cancer Institute Emory University"
+# Organization: Australian Prostate Cancer Research Centre
+msgid "Australian Prostate Cancer Research Centre"
 msgstr ""
 
-# Intervention: care_plan
-msgid "<p>Organization and support for the many details of life as a prostate cancer survivor</p>"
-msgstr "<p>Organización y apoyo para los múltiples detalles de la vida como sobreviviente del cáncer de próstata</p>"
-
```
Comment (mcjustin) [inline portal/translations/es_ES/LC_MESSAGES/messages.po]: still a bunch of deletions, are they ok?

Comment (ivan-c) [reply portal/translations/es_ES/LC_MESSAGES/messages.po]: Looks like that only got moved to [here](https://github.com/uwcirg/truenth-portal/pull/3185/files/2c9a998ad5f66275a803e78225eeaba67a10a9b1#diff-68c37b10c885be7e2fa3dee8ff2d5759R3198)

If you hover over header for each file you can see the insertions/deletions; I skimmed them and roughly all of them have net insertions

image

File: portal/translations/es_ES/LC_MESSAGES/messages.po (hunk 1 above)
Comment (ivan-c) [reply portal/translations/es_ES/LC_MESSAGES/messages.po]: Probably not translated (authorized) yet; I'll make sure to add it when I go through the strings needing authorization

**Pull request merged**